src/components/ProcessingPage/EditingPage/index.jsx

```diff
@@ -18,12 +21,20 @@ function useProgressStatus() {
     },
     // mockup시연의 경우 짧게 설정하고, 이후는 5-10초로 설정합니다.
     refetchInterval: 1000,
+    enabled: videoStatus,
   });
 }
 
 function EditingPage() {
-  const { data, isLoading, isError } = useProgressStatus();
-  // TODO. 서버의 진행상황을 props로 ProgressBox, ProgressBar에 전달해야합니다.
+  const [isCuttingInProgress, setIsCuttingInProgress] = useState(true);
+  const { finalVideoUrl } = useFinalVideoUrlStore();
+  const { data } = useProgressStatus(isCuttingInProgress);
+
+  useEffect(() => {
+    if (finalVideoUrl !== "") {
+      setIsCuttingInProgress(false);
+    }
```

Comment:
```suggestion
    if (finalVideoUrl.length) {
      setIsCuttingInProgress(false);
    }
```
또는 
    if (!finalVideoUrl) {
      setIsCuttingInProgress(false);
    }
```
처럼 써도 좋을 것 같다는 의견입니다! 하지만 지금도 명시적이라 좋네요!

Reply:
강제 형 변환은 최대한 지양하는 것이 좋다고 생각하는데 수진님 의견은 어떠신가요? 스크럼 회의때 이야기를 나누어봐도 좋겠네요!

Reply:
확인했습니다 :) 이대로 가도 좋을 것 같아요!